### PR TITLE
Ceres: Fix CMakeLists.txt, add libconfig dependency

### PR DESCRIPTION
**Pull request by tamiko**
This are a couple of small CMake fixes to make this code-gallery project
compile.

### CODE REVIEW

File: CeresFE/CMakeLists.txt [hunk 1]
```diff
@@ -2,41 +2,33 @@
...
-#  FILE(GLOB_RECURSE TARGET_INC  "include/*.h")
-#  SET(TARGET_SRC ${TARGET_SRC}  ${TARGET_INC})
-# or switch altogether to the large project CMakeLists.txt file discussed
-# in the "CMake in user projects" page accessible from the "User info"
-# page of the documentation.
-SET(TARGET_SRC
-  ${TARGET}.cc
+CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)
+
+FIND_PACKAGE(deal.II 8.3 REQUIRED
```
Comment (bangerth) [inline CeresFE/CMakeLists.txt]: Just ask for 9.0.

**Comment by masterleinad** [comment]
I noticed a few more things that needed to be fixed an created a PR(tamiko/code-gallery#1) to this PR.
In particular, I ask there for `deal.II-9.0`.

**Comment by tamiko** [comment]
The PR is now updated.

**Comment by tamiko** [comment]
> The libraries `libconfig` build for me is `liblibconfig.so` and
> `liblibconfig++.so`. Maybe, we should accept both `config.so` and
> `libconfig.so`?

Then let's accept both. Who on earth garbles up the library as "liblibconfig++.so". This
is terrible.

File: CeresFE/CMakeLists.txt [hunk 2]
```diff
@@ -2,30 +2,12 @@
...
-  ${TARGET}.cc
-  )
-
-# Usually, you will not need to modify anything beyond this point...
-
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)
 
-FIND_PACKAGE(deal.II 8.3 QUIET
+FIND_PACKAGE(deal.II 9.0 SILENT
```
[the remark this answers is not in the document]
Comment (masterleinad) [reply CeresFE/CMakeLists.txt]: @tamiko ping

**Pull request merged**